python/cudf/cudf/_lib/pylibcudf/hashing.pyx

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+from libcpp.memory cimport unique_ptr
+from libcpp.utility cimport move
+
+from cudf._lib.cpp.hash cimport(
+    murmurhash3_x86_32 as cpp_murmurhash3_x86_32,
+    murmurhash3_x64_128 as cpp_murmurhash3_x64_128,
+    md5 as cpp_md5,
+    sha1 as cpp_sha1,
+    sha224 as cpp_sha224,
+    sha256 as cpp_sha256,
+    sha384 as cpp_sha384,
+    sha512 as cpp_sha512,
+    xxhash_64 as cpp_xxhash_64
+)
+from cudf._lib.cpp.column.column cimport column
+from cudf._lib.cpp.table.table cimport table
+from libc.stdint cimport uint32_t, uint64_t
+
+from .table cimport Table
+from .column cimport Column
+
+cpdef Column murmurhash3_x86_32(
+    Table input, 
+    uint32_t seed
+):
+    cdef unique_ptr[column] c_result
+    with nogil:
+        c_result = move(
+            cpp_murmurhash3_x86_32(
+                input.view(),
+                seed
+            )
+        )
+
+    return Column.from_libcudf(move(c_result))
+
+cpdef Table murmurhash3_x64_128(
+    Table input,
+    uint64_t seed
+):
+    cdef unique_ptr[table] c_result
+    with nogil:
+        c_result = move(
+            cpp_murmurhash3_x64_128(
+                input.view(),
+                seed
+            )
+        )
+
+    return Table.from_libcudf(move(c_result))
+
+
+cpdef Column xxhash_64(
+    Table input, 
+    uint64_t seed
+):
+    cdef unique_ptr[column] c_result
+    with  nogil:
+        c_result = move(
+            cpp_xxhash_64(
+                input.view(),
+                seed
+            )
+        )
+
+    return Column.from_libcudf(move(c_result))
+
+
```

Comment:
I know there's no Cython formatting set up, but could you take a pass over this file and try to make the spacing consistent? The number of blank lines between functions is not consistent.

Reply:
I think most of the Cython files aim for two empty lines between functions, but please double-check that and match the convention from other files.